storm-client/src/jvm/org/apache/storm/Config.java

```diff
@@ -372,9 +374,10 @@
     /**
      * How many executors to spawn for ackers.
      *
-     * <p>By not setting this variable or setting it as null, Storm will set the number of acker executors
-     * to be equal to the number of workers configured for this topology. If this variable is set to 0,
-     * then Storm will immediately ack tuples as soon as they come off the spout, effectively disabling reliability.</p>
+     * <p>By not setting this variable or setting it as null, Storm will set the number of acker executor to be equal to
```

Comment:
nit: please restore pluralization of `executor`.  i.e., `number of acker executors`.